<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QMainWindow" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">QGroupBox{
border: 0;
}
QComboBox{
border-radius: 5px;
padding: 5px;
color: gray;
}
QComboBox::drop-down { 
	width: 0px; 
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="logo">
         <property name="minimumSize">
          <size>
           <width>261</width>
           <height>111</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>261</width>
           <height>111</height>
          </size>
         </property>
         <property name="pixmap">
          <pixmap resource="Resource.qrc">:/images/Grubway.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sign Up</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="firstNameInput">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
    padding: 10px;
    border-radius: 5px;
    }</string>
         </property>
         <property name="placeholderText">
          <string>First Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lastNameInput">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
    padding: 10px;
    border-radius: 5px;
    }</string>
         </property>
         <property name="placeholderText">
          <string>Last Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="addressInput">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
    padding: 10px;
    border-radius: 5px;
    }</string>
         </property>
         <property name="placeholderText">
          <string>Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="emailInput">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
    padding: 10px;
    border-radius: 5px;
    }</string>
         </property>
         <property name="placeholderText">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="phoneNumberInput">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
    padding: 10px;
    border-radius: 5px;
    }</string>
         </property>
         <property name="placeholderText">
          <string>Phone Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="usernameInput">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
    padding: 10px;
    border-radius: 5px;
    }</string>
         </property>
         <property name="placeholderText">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordInput">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
    padding: 10px;
    border-radius: 5px;
    }</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="placeholderText">
          <string>Position</string>
         </property>
         <item>
          <property name="text">
           <string>Cashier</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Manager</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Administrator</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="code">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
    padding: 10px;
    border-radius: 5px;
    }</string>
         </property>
         <property name="placeholderText">
          <string>Administrator Code</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RegisterButton">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <italic>false</italic>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
    color: white;
    background-color: #0e67ed;
    border-radius: 5px;
    padding: 5px;
    }</string>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>689</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
